_drafts/device-logging.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Trade-offs and considerations for logging on embedded devices
```

Comment:
Title doesn't seem consistent with the rest of our articles.

Reply:
years of square has made me prefer sentence casing to title casing 🙈